timemachine/fe/rest/single_topology.py

```diff
@@ -117,7 +117,7 @@ def get_energy_scale_factor(self, lamb: float) -> float:
         temperature_factor = self._temperature_scale_interpolation_fxn(lamb).item()
         return 1.0 / temperature_factor
 
-    def setup_intermediate_state(self, lamb: float) -> VacuumSystem[NonbondedPairListPrecomputed, HarmonicAngleStable]:
```

Comment:
I guess this is one of the areas where we "lose" the additional "specificity" of the type, since now we return something more generic/encompassing.

Reply:
stale comment since we brought back specific systems